pkg/client/factory.go

```diff
@@ -81,12 +103,29 @@ func (f *factory) BindFlags(flags *pflag.FlagSet) {
 	flags.AddFlagSet(f.flags)
 }
 
-func (f *factory) Client() (clientset.Interface, error) {
+func (f *factory) ClientConfig() (*rest.Config, error) {
 	clientConfig, err := Config(f.kubeconfig, f.kubecontext, f.baseName)
 	if err != nil {
 		return nil, err
 	}
 
+	if f.clientQPS > 0.0 {
```

Comment:
should we just pass `clientQPS` and `clientBurst` into the `Config(...)` function as arguments?

Reply:
Right now they're only on the `server` subcommand, nothing else uses them. So we'd likely want to promote them to be directly on `velero`.

I can see them being useful for `run-plugins`, though maybe not user-facing commands.

Reply:
I mean make L109 `clientConfig, err := Config(f.kubeconfig, f.kubecontext, f.baseName, f.clientQPS, f.clientBurst`)`.  Seems arbitrary to not have them set within the `Config` func itself.  I don't think we need to change any CLI commands?

Reply:
Ah, I see. Yes, that makes sense.